reference-implementation/lib/helpers.js

```diff
@@ -62,6 +62,10 @@ exports.IsFiniteNonNegativeNumber = v => {
 };
 
 exports.InvokeOrNoop = (O, P, args) => {
+  assert(exports.typeIsObject(O) === true);
+  assert(typeof P === 'string');
+  assert(Array.isArray(args));
```

Comment:
I just noticed that, but... why does it say that? Every single caller passes a List, and this op isn't exported.